builder/azure/common/client/config.go

```diff
@@ -224,9 +217,9 @@ func (c Config) Validate(errs *packersdk.MultiError) {
 		if err != nil {
 			errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("client_jwt is not a JWT: %v", err))
 		} else {
-			if claims.ExpiresAt < time.Now().Add(5*time.Minute).Unix() {
-				errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("client_jwt will expire within 5 minutes, please use a JWT that is valid for at least 5 minutes"))
-			}
+			//if claims.ExpiresAt < time.Now().Add(5*time.Minute).Unix() {
+			//	errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("%d %d, client_jwt will expire within 5 minutes, please use a JWT that is valid for at least 5 minutes", claims.ExpiresAt, time.Now().Add(5*time.Minute).Unix()))
+			//}
```

Comment:
Is there a reason why this gets commented and not removed?

Reply:
Good catch Lucas! This check should be removed as it prevents OIDC authentication, I had commented it out earlier while testing OIDC and forgot to remove it, I'll do so